src/main/java/org/json/JSONObject.java

```diff
@@ -1762,12 +1762,7 @@ private void populateMap(Object bean, Set<Object> objectsRecord) {
         Method[] methods = includeSuperClass ? klass.getMethods() : klass.getDeclaredMethods();
         for (final Method method : methods) {
             final int modifiers = method.getModifiers();
```

Comment:
The variable `modifiers` is only used with the method `isValidMethod`. So there is no need for this local variable here.
Instead, it should be a variable inside the method `isValidMethod` and the parameter should be removed from the method signature.